plugins/SolidView/SolidView.py

```diff
@@ -111,7 +120,18 @@ def _onPropertyChanged(self, key: str, property_name: str) -> None:
             else:
                 angle = support_angle_stack.getProperty("support_angle", "value")
                 if angle is not None:
-                    self._support_angle = angle
+                    return angle
+        return 90.0
+
+    def _retrieveLowestPrintHeight(self) -> float:
+        min_height = 0.0
+        for extruder in Application.getInstance().getExtruderManager().getActiveExtruderStacks():
+            init_layer_height = extruder.getProperty("layer_height_0", "value")
+            tolerance_setting = extruder.getProperty("slicing_tolerance", "value")
+            if tolerance_setting == "middle":
+                init_layer_height /= 2.0
+            min_height = max(min_height, init_layer_height / 2.0)
```

Comment:
This calculation is incorrect. It's dividing the initial layer height by 2 twice. As a result, the height at which it shows inverted colours is a quarter of the initial layer height by default, rather than half.

To test it I used a staircase model with 10mm steps, and an initial layer height of 21mm. It showed that the base step of the stair would be printed as normal.
![image](https://user-images.githubusercontent.com/2448634/109833116-4a86d180-7c41-11eb-9cc3-ff8063791772.png)

While actually it was missing the first step since the cross section is taken at 10.5mm height:
![image](https://user-images.githubusercontent.com/2448634/109833306-72763500-7c41-11eb-967d-ee0b81d7cd66.png)

So I'll remove this second division by 2 here. I hope you'll agree @rburema .

Reply:
If you think it then works for all tolerances, please do! :-)